src/utils/proxy.js

```diff
@@ -70,7 +70,7 @@ const proxyToExternalUrl = function ({ dest, destURL, req, res }) {
     pathRewrite: () => destURL,
     ...(Buffer.isBuffer(req.originalBody) && { buffer: toReadableStream(req.originalBody) }),
   })
-  return handler(req, res, {})
+  return handler(req, res, () => {})
```

Comment:
I only noticed this because a request was not matching and http-proxy-rewrite was actually calling `next` (the third parameter)

```
next is not a function
```